### ISSUE TITLE
Using GameFinder to locate DBD

### ISSUE THREAD

**Issue by ray1997** [Charzard4261/DBD-ResourcePackManager#1]
It might help check a bit if they got a game install anywhere either on Steam or Epic, and set it for them on the first launch?

**Comment by Charzard4261** [comment]
That's a good idea! I don't have it on Epic, do you know what the default install path is for it?

**Comment by ray1997** [comment]
> That's a good idea! I don't have it on Epic, do you know what the default install path is for it?

I used to test its a long time ago, it has a long ID of "611482b8586142cda48a0786eb8a127c:467a7bed47ec44d9b1c9da0c2dae58f7:Brill" but I'm not sure if it's gonna work. Anyway, PR on the way

Additionally you could enumerate through all installed egs games:
```
        foreach (var (game, error) in handler.FindAllGames())
        {
            if (game is not null)
            {
                //There's a better way than this, feel free to change
                bool d = game.DisplayName.ToLower().Contains("dead");
                bool b = game.DisplayName.ToLower().Contains("by");
                bool dd = game.DisplayName.ToLower().Contains("daylight");
                if (d && b && dd)
                {
                    return game.InstallLocation;
                }
            }
        }
```

**Comment by Charzard4261** [comment]
Thanks for doing that! Will be released with version 0.9.8.
